src/js/components/VoterGuide/EditPositionAboutCandidateModal.jsx

```diff
@@ -84,17 +84,18 @@ export default class EditPositionAboutCandidateModal extends Component {
   }
 
   _onSupportStoreChange () {
-    var candidate_we_vote_id = this.props.position.ballot_item_we_vote_id;
-    this.setState({ supportProps: SupportStore.get(candidate_we_vote_id) });
+    var ballot_item_we_vote_id = this.props.position.ballot_item_we_vote_id;
```

Comment:
seems like most of these changes are designed to support all ballot items (including measures), but I notice the filename is EditPositionAbout**Candidate**Modal.  Are these changes merely an effort to streamline/standardize naming conventions, or is this file meant to handle measures too?